portal-web/test/functional/com/liferay/portalweb/paths/portlet/marketplace/AppManager.path

```diff
@@ -26,6 +26,92 @@
 	<td></td>
 	<td></td>
 </tr>
+<!--APP_STORE-->
+<tr>
+	<td>APP_FILTER_LABEL</td>
+	<td>//span[contains(@class,'aui-field-select')]/span[contains(@class,'aui-field-content')]/label[contains(.,'${key_filterName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_FILTER_SELECT_DROP_DOWN_LIST</td>
+	<td>//label[contains(.,'${key_filterName}')]/following-sibling::span/select</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_FILTER_SELECTED_OPTION</td>
+	<td>//span[contains(@Class,'aui-field-content')]//select/option[contains(.,'${filterBy}') and @selected]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_PURCHASE</td>
+	<td>//div[contains(@class,'purchase')]/a[contains(.,'${key_type}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_SUBMIT</td>
+	<td>//input[contains(@type,'submit') or contains(@class,'submit')]</td>
```

Comment:
Update the locator to //input[contains(.,'submit')] as you said in https://github.com/Sharryxue/liferay-portal/pull/347/files#r832935358.

Reply:
done

Reply:
//input[contains(.,'submit') dosen't work. so I didn't update it. and Done